src/controllers/graphql/resolvers/post/postResolver.js

```diff
@@ -21,14 +21,21 @@ module.exports = {
     },
     getHundredxPosts: async (_, args) => {
       return await PostService.fetchHundredxResteemedPosts();
-    }
+    },
+  },
+  Mutation: {
+    broadcastPost: async (_, { permLink, title, body, tags }, { authenticatedUserInstance } ) => {
+      return !authenticatedUserInstance
```

Comment:
Let's use an `if` statement here
```
broadcastPost: async() => {
   if (!authenticatedUserInstance) {
      return new AuthenticationError('');
   }
   await PostService.callFunction();
}
```

Reply:
Ternaries should be used for very simple operations. A general rule of thumb is if you have to break apart your ternary into multiple lines, you should probably be using an `if` statement

Reply:
We're also using a ternary for error checking in `getFollowerPosts`. Could we change that to an `if statement as well?

Reply:
sure